int16 left_hz
int16 right_hz
uint32 duration_ms
---
bool sucess
